package exception;

public class Test3 {
	public static void main(String[] args) throws ArithmeticException,InterruptedException  {
		System.out.println(1);
		System.out.println(2);
		System.out.println(3);
		System.out.println(4/0);//this is not posible
		System.out.println(5);
		System.out.println(6);
		System.out.println(7);
		
	}

}
